lib/mix/tasks/compile.make.ex

```diff
@@ -83,6 +83,11 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   and `mix test` commands.
   """
 
+  @exec_notfound_msg """
```

Comment:
The reason is that they are big error messages and I didn't want them to impact the code readability (seeing something like 10 lines of prose in the middle of some code kinda distracts from the purpose of the code).